### PR TITLE
Bumbed node version to 7.7.3

### LINKED ISSUE #29: Node >= 7.6 support
Our node project requires node >= 7.6, it is currently not available in registry.opensource.zalan.do

### PR DESCRIPTION
**Pull request by semonte**
Also updated GPG keys to match with keys found at
https://github.com/nodejs/docker-node/blob/master/7.7/wheezy/Dockerfile

Fixes #29

### CODE REVIEW

File: stable/Dockerfile [hunk 1]
```diff
@@ -10,18 +10,18 @@ RUN set -ex \
...
     C4F0DFFF4E8C1A8236409D08E73BC641CC11F4C8 \
+    56730D5401028683275BD23C23EFEFE93C4CFFFE \
   ; do \
     gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
   done
 
 ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 7.4.0
+ENV NODE_VERSION 7.7.3
 ENV NPM_VERSION 4.1.2
```
Comment (prayerslayer) [inline stable/Dockerfile:25]: Let's bump this to `4.4.3` also :)

**Comment by prayerslayer** [comment]
Looks good, thanks a lot! I'd appreciate it if you could upgrade the NPM version too.

File: stable/Dockerfile (hunk 1 above)
Comment (semonte) [reply stable/Dockerfile:25]: Ok, I did not update that because npm 4.1.2 is listed at https://nodejs.org/en/download/releases/ with node 7.7.3.

**Comment by semonte** [comment]
I tried building with npm 4.4.3 but it fails:

```
npm ERR! Linux 4.9.8-moby
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "npm@4.4.3"
npm ERR! node v7.7.3
npm ERR! npm  v4.1.2
npm ERR! path /usr/local/lib/node_modules/.staging/@npmcorp/copy-5f73018a
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename

npm ERR! enoent ENOENT: no such file or directory, rename '/usr/local/lib/node_modules/.staging/@npmcorp/copy-5f73018a' -> '/usr/local/lib/node_modules/npm/node_modules/@npmcorp/copy'
npm ERR! enoent ENOENT: no such file or directory, rename '/usr/local/lib/node_modules/.staging/@npmcorp/copy-5f73018a' -> '/usr/local/lib/node_modules/npm/node_modules/@npmcorp/copy'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     /npm-debug.log
```

**Comment by semonte** [comment]
There is an open issue about this in https://github.com/npm/npm/issues/16055 So I would not update the npm version at this point.

**Comment by prayerslayer** [comment]
Thanks for checking, I would have blindly updated. Then let's merge this as-is.

**Pull request merged**